style.css

```diff
@@ -0,0 +1,3 @@
+* {
```

Comment:
You could have added more styling to improve the User experience, eg it is good practice to use the same styling for the same tags such as the h1 tags in your code.